scripts/memory-scorecard.sh

```
@@ -0,0 +1,335 @@
+    export PATH="$BIN_DIR:$PATH"; cd "$proj"
+    ${SESSION_TIMEOUT} claude -p "$prompt" \
+      --setting-sources project --model "$MODEL" --max-budget-usd "$MAX_BUDGET_USD" \
+      --permission-mode acceptEdits --allowedTools "Bash Edit Write" "$@" \
+      >"$log" 2>&1 || true
+  )
+}
+
+install_rusty_brain() { # home proj
+  local home="$1" proj="$2"
+  (
+    export HOME="$home"; export PATH="$BIN_DIR:$PATH"; cd "$proj"
```

Shellcheck (0.11.0)

[info] Modification of HOME is local (to subshell caused by (..) group).

(SC2030)